scripts/cms5/keepalive.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+// run on a day of the month up to the 28th
+$runOnDay = predictable_random_int(27);
```

Comment:
```suggestion
// run on a day of the month up to the 28th
$runOnDay = predictable_random_int(28);
```
And make sure it can't be 0

Reply:
Updated